### PR TITLE
Retrieve endpoint/15

### PR DESCRIPTION
**Pull request by hariharansrc**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -7,35 +7,40 @@ scalaVersion := "2.12.4"
 val akkaHttpVersion = "10.1.5"
 
 libraryDependencies += "org.parboiled" %% "parboiled" % "2.1.4"
-libraryDependencies += "com.typesafe.akka" %% "akka-http" % akkaHttpVersion
+
+libraryDependencies += "com.typesafe.akka" %% "akka-http" % "10.0.11"
+libraryDependencies += "com.typesafe.akka" %% "akka-http-testkit" % "10.0.11"
```
Comment (bhermann) [inline build.sbt]: Why did you remove the usage of `akkaHttpVersion` and reverted back to the old (and vulnerable) version of `akka-http`?

Comment (hariharansrc) [reply build.sbt]: It was a merge conflict. Reverted to the correct http version.

**Comment by hariharansrc** [comment]
Since tests needs elasticsearch and webapi running. `ElasticActorTest` is moved to it(IntegrationTest) folder. It can be run using `it:test` in sbt.
